### ISSUE TITLE
UI Improvements

### ISSUE THREAD

**Issue by vinitshahdeo** [vinitshahdeo/online-debate-system#3]
> Currently the homepage UI seems dull, using basic CSS styling. Would be nice to have a better design. 

- Create an **eye-catching** layout for the homepage.
-  Do not make changes in other files unnecessarily.
- **Feel free to update CSS** (You may use any library say [Bootstrap](https://getbootstrap.com/)).
- Kindly provide screenshots of UI changes in your PR description.

Thanks! 😃 

[![Open Source Love](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://github.com/vinitshahdeo)

**Comment by Exoceus** [comment]
I'll work on it!

**Comment by ghost** [comment]
Can more than one person submit a PR for this issue? I would be interested in working on this as well. Thanks!

**Comment by saltgen** [comment]
I'll work on this, cheers!

**Comment by vinitshahdeo** [comment]
@codetamaracode Yeah, please go ahead. I'll be merging the best one. Give it a shot.

**Comment by vinitshahdeo** [comment]
@saltgen Wow, please proceed.

**Comment by Siddhant-K-code** [comment]
working on UI/UX of start button

**Comment by vinitshahdeo** [comment]
## CHECK OUT [THIS LINK](https://corona-cases-india.netlify.com/) FOR LIVE CORONA UPDATES

Hello everyone,

I hope you are staying safe at your home and enjoying the contribution to your awesome open source projects. **I've created a web app to display the latest corona updates, please check [here](https://corona-cases-india.netlify.com/).**

#### :wave: [DO THE FIVE](https://www.mohfw.gov.in/): Help stop coronavirus

1. **HANDS**: Wash them often
2. **ELBOW**: Cough into it
3. **FACE**: Don't touch it
4. **SPACE**: Keep safe distance
5. **HOME**: Stay if you can

#### [COVID-19](https://corona-cases-india.netlify.com/): Stay Home<sup>Stay Safe</sup>

- Avoid Handshakes 🤝 ❌
- Do Namastey 🙏 ✔️


---

<sup>**PS: I know it's off-topic but **Yes**, COVID-19 everyone's talking about. Please stay safe at your home and enjoy contributing to your projects.** </sup>

**Comment by vinitshahdeo** [comment]
### Hello everyone,

> # Do check out my latest [COVID-19 😷Tracker](https://indiafightscorona.netlify.app/) | INDIA 🇮🇳

[![COVID-19 Tracker by Vinit Shahdeo](https://user-images.githubusercontent.com/20594326/79394735-d1817b80-7f95-11ea-90a4-7b99e86f8560.png)](https://indiafightscorona.netlify.app/)

---
<sup><strong> I've just revamped the UI(WIP). You can check the old website [here](https://corona-cases-india.netlify.app/). Consider leaving a star [here](https://github.com/vinitshahdeo/COVID19).  :star: <br>(To be open-sourced pretty soon.)</strong></sup>

[![GitHub stars - COVID-19 Vinit Shahdeo](https://img.shields.io/github/stars/vinitshahdeo/COVID19?label=LEAVE%20A%20Star%20on%20GitHub&logo=github&style=for-the-badge)](https://github.com/vinitshahdeo/COVID19/)

**Comment by dipenr06** [comment]
@vinitshahdeo can i take up this issue please?

**Comment by satkar2001** [comment]
@vinitshahdeo  can i work on the button UI

**Comment by tanishagupta1** [comment]
Can I add basic instructions as in what are the steps a user must follow after clicking the start button, It is a bit unclear as the PHP file gets downloaded without knowing what has to be done with it.

**Comment by mitalrs** [comment]
Can 

> > Currently the homepage UI seems dull, using basic CSS styling. Would be nice to have a better design.
> 
> * Create an **eye-catching** layout for the homepage.
> * Do not make changes in other files unnecessarily.
> * **Feel free to update CSS** (You may use any library say [Bootstrap](https://getbootstrap.com/)).
> * Kindly provide screenshots of UI changes in your PR description.
> 
> Thanks! smiley
> 
> [![Open Source Love](https://camo.githubusercontent.com/709f3cbabb06b8896edf6890835a638693bb9f70e6002a959e189190d55d6111/68747470733a2f2f6261646765732e66726170736f66742e636f6d2f6f732f76322f6f70656e2d736f757263652e7376673f763d313033)](https://github.com/vinitshahdeo)

Can I also work on this project?.... I'm also interested!

**Comment by the-eduek** [comment]
Hi @vinitshahdeo! I would love to work on this issue

**Comment by sudip-101** [comment]
Hi @vinitshahdeo ,
I would like to work on this issue.
Should I go ahead or is it already done?
